metadata-db-app/src/cmr/metadata_db/data/oracle/generic_documents.clj

```diff
@@ -7,16 +7,14 @@
    ;[cmr.common.log :refer [debug info warn error]]
    [cmr.common.time-keeper :as tkeep]
    [cmr.common.util :as cutil]
-   ;[cmr.metadata-db.data.oracle.concepts :as concepts]
-   ;[cmr.metadata-db.data.oracle.concept-tables :as ct]
    [cmr.metadata-db.data.oracle.sql-helper :as sh]
    [cmr.metadata-db.data.generic-documents :as gdoc]
    [cmr.oracle.sql-utils :as su :refer [insert values select from where with order-by desc
                                         delete as]]
    [cmr.oracle.connection :as oracle]
    [clj-time.coerce :as coerce]
    [cmr.common.date-time-parser :as dtp]
-   ;[clojure.java.io :as io]
+   [cmr.common.util :as cutil]
    )
```

Comment:
This needs to be moved up so that the list is in alphabetical order.

Reply:
installed a plugin for sorting lines of code